schemas/common_definitions.json

```diff
@@ -301,5 +301,11 @@
   "non_empty_string": {
     "type": "string",
     "minLength": 1
+  },
+  "string_list": {
```

Comment:
As all the places this is used it's replacing `string_with_placeholders`, we should consider naming it `string_list_with_placeholders` and putting it in the string interpolation definitions file.

Reply:
Or `string_array_with_placeholders` given it's JSON not Python?